airflow/decorators/__init__.pyi

```diff
@@ -0,0 +1,105 @@
+from typing import Optional, Callable, Iterable, Dict, Union, List
+
+try:
+    from airflow.providers.docker.decorators.docker import DockerDecoratorMixin
+except ImportError:
+    DockerDecoratorMixin = None
+
+class _TaskDecorator:
+    def __call__(self, *args, **kwargs):
+        try:
+            from airflow.providers.docker.decorators import docker
+
+        except ImportError:
+            pass
+    def python(
```

Comment:
@ashb and I spoke it over and for now we're just going to keep python and virtualenv in the taskdecorator class. It doesn't change anything functionally and unlike external providers, python and virtualenv will ONLY update with updates to Airflow.

Reply:
Makes perfect sense!